### ISSUE TITLE
[A11y] Dashboard - Hover - Graphs

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1858]
### Operating System

Windows

### Browser

Edge

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Hovering fails in the following ways:

- The tooltips aren't dismissable without moving the mouse
- Users cannot move the mouse into the tooltip (for instance to copy and paste) because even a tiny movement in the mouse will switch to a new tooltip

### Suggested Fix

This is a WCAG 2.1 AA standard: https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html.  I mention this because I didn't realize this was a AA criteria in the past.

- Implement the escape key dismissing the tooltips
- Make sure tooltips stay available if the user is trying to move the mouse within one

**Comment by azle** [comment]
Update 1/6/2024

Still an issue and when a user tabs it skips the whole repository growth and visibility section and goes directly to Admin Sets

<img width="1084" alt="Screenshot 2025-01-06 at 1 38 16 PM" src="https://github.com/user-attachments/assets/3490e746-b211-4b86-869a-8a3b0a8dd7fc" />
